Add Kubernetes process detector support (#16)
apache/skywalking-rover: docs/en/setup/configuration/process_discovery/kubernetes.md

| process_discovery.kubernetes.namespaces | | ROVER_PROCESS_DISCOVERY_KUBERNETES_NAMESPACES | Including pod by namespaces, if empty means including all namespaces. Multiple namespaces split by ",". |
| process_discovery.kubernetes.analyzers | | | Declare how to build the process. The istio and k8s resources are active by default. |
| process_discovery.kubernetes.analyzers.active | | | Set is active analyzer. |
| process_discovery.kubernetes.analyzers.filters | | | Define which process is match to current process builder. |
| process_discovery.kubernetes.analyzers.service_name | | | The Service Name of the process entity. |
| process_discovery.kubernetes.analyzers.instance_name | | | The Service Instance Name of the process entity, by default, the instance name is the host IP v4 address from "en0" net interface. |
| process_discovery.kubernetes.analyzers.process_name | | | The Process Name of the process entity, by default, the process name is the executable name of the process. |
| process_discovery.kubernetes.analyzers.labels | | | The Process Labels, used to aggregate similar process from service entity. Multiple labels split by ",". |

## Process Analyze

> The process analyze is declare which process could handle and how to build the process entity. 

Reviewer: ```suggestion
The process analyze declares which process could be profiled and how to build the process entity. 
```
